Cython/Compiler/Symtab.py

```diff
@@ -2136,7 +2136,7 @@ def needs_tp_clear(self):
 
     def mangle_class_private_name(self, name):
         # a few utilitycode names need to specifically be ignored
-        if name and name.startswith("__pyx_unpickle_"):
+        if name and name.lower().startswith("__pyx"):
```

Comment:
Appending at least another underscore should make it a bit safer.
```suggestion
        if name and name.lower().startswith("__pyx_"):
```